openrtb_ext/bidders.go

```diff
@@ -255,6 +255,7 @@ const (
 	BidderYieldmo           BidderName = "yieldmo"
 	BidderYieldone          BidderName = "yieldone"
 	BidderZeroClickFraud    BidderName = "zeroclickfraud"
+	BidderUndertone         BidderName = "undertone"
```

Comment:
Preference here is to add builder in ascending order. So should move `BidderUndertone BidderName = "undertone"` on line 242